packages/core/types/SlackInteractionRequest.ts

```diff
@@ -5,7 +5,15 @@ export const birthdayConfirmActionId = "birthdayConfirm";
 export const birthdayIncorrectActionId = "birthdayIncorrect";
 export const presentIdeasInputActionId = "presentIdeas";
 export const presentIdeasInputBlockId = "presentIdeasInput";
```

Comment:
Could you please rename these 2 as well? Your naming convention is much clearer and better.